openapi-cli/src/main/java/org/ballerinalang/ballerina/OpenApiConverterUtils.java

```diff
@@ -0,0 +1,417 @@
+
+/**
+ * OpenApi related utility classes.
+ */
+
+public class OpenApiConverterUtils {
+
+    private static SyntaxTree syntaxTree;
+    private static SemanticModel semanticModel;
+    private static Project project;
+    private static List<ListenerDeclarationNode> endpoints = new ArrayList<>();
+
+
+    /**
+     * This util for generating files when not available with specific service name.
+     * @param servicePath
+     * @param outPath
+     * @throws IOException
+     * @throws OpenApiConverterException
+     * @return
+     */
+    public static void  generateOAS3DefinitionsAllService(Path servicePath, Path outPath, String serviceName)
+            throws IOException, OpenApiConverterException {
+
+
+        // Load project instance for single ballerina file
+        project = SingleFileProject.load(servicePath);
+        //travers and filter service
+
+        // Take package name for project
+        Package packageName = project.currentPackage();
+        // Take module instance for traversing the syntax tree
+        Module currentModule = packageName.getDefaultModule();
+        semanticModel =  currentModule.getCompilation().getSemanticModel();
+        Iterator<DocumentId> documentIterator = currentModule.documentIds().iterator();
+        while (documentIterator.hasNext()) {
+            DocumentId docId = documentIterator.next();
+            Document doc = currentModule.document(docId);
+            syntaxTree = doc.syntaxTree();
+            ModulePartNode modulePartNode = syntaxTree.rootNode();
+            for (Node node : modulePartNode.members()) {
+                SyntaxKind syntaxKind = node.kind();
+                // Load a listen_declaration for the server part in the yaml spec
+                if (syntaxKind.equals(SyntaxKind.LISTENER_DECLARATION)) {
+                    ListenerDeclarationNode listener = (ListenerDeclarationNode) node;
+                    endpoints.add(listener);
+                }
+                // Load a service Node
+                if (syntaxKind.equals(SyntaxKind.SERVICE_DECLARATION)) {
+                    ServiceDeclarationNode serviceNode = (ServiceDeclarationNode) node;
+                    NodeList<Node> serviceNameNodes = serviceNode.absoluteResourcePath();
+                     String openApiName = getOpenApiFileName(syntaxTree.filePath(), serviceName);
+                     String ballerinaSource = readFromFile(servicePath);
+                     String openApiSource = generateOAS3Definitions(syntaxTree, serviceName);
+
+                    //  Checked old generated file with same name
+                    openApiName = checkDuplicateFiles(outPath, openApiName);
+                    writeFile(outPath.resolve(openApiName), openApiSource);
+                }
+            }
+        }
+    }
+
+    private static String getOpenApiFileName(String servicePath, String serviceName) {
+        String openApiFile;
+
+        if (StringUtils.isNotBlank(serviceName) || serviceName != null) {
```

Comment:
Java 11 has added a new instance method `isBlank()` to `java.lang.String class` so you don't have to use apache libraries in here.